inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cmd/shell/ShellExecutorImpl.java

```diff
@@ -92,11 +93,26 @@ private static boolean HasException(String str) {
         return false;
     }
 
+    public void syncExec(String shellPath, boolean includeEmptyParams, String... params) {
```

Comment:
Why add this parameter, and it is always true？

Reply:
Based on the current design of the `execRemote` method, it calls the expect script using hard-coded values. This script receives a password parameter. If the password parameter is empty, it will be removed by the original merge method, causing parameter mapping errors in the expect script. Currently, the SSH key authentication method does not require a password, so it is indeed necessary to retain the empty string parameter when invoke the exec_cmd.exp script.

Reply:
If the string is allowed to be empty, then modify the deletion logic of the original merge method instead of setting this parameter.

Reply:
Okay, I will remove the logic that deletes empty parameters. If we need to delete parameters in the future, we can make the corresponding changes then